src/main/java/de/tum/cit/aet/artemis/nebula/dto/FaqRewritingResponse.java

```diff
@@ -0,0 +1,6 @@
+package de.tum.cit.aet.artemis.nebula.dto;
+
+import jakarta.validation.constraints.NotNull;
+
+public record FaqRewritingResponse(@NotNull String rewrittenText) {
```

Comment:
Why don't you call this DTO a DTO?
```suggestion
public record FaqRewritingResponseDTO(@NotNull String rewrittenText) {
```